packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -276,6 +329,53 @@ const Updates = decorate([
             </CardBlock>
           </Card>
         </Col>
+        <Col mediumSize={6}>
+          <Card>
+            <CardHeader>{_('releaseChannels')}</CardHeader>
+            <CardBlock>
+              <form id='releaseChannelsForm' className='form'>
```

Comment:
Never use static identifiers in React, generate an id via a computed if necessary.

Reply:
why via a `computed` ?

Reply:
IMHO, it's the easiest way: https://github.com/vatesfr/xen-orchestra/blob/8210fddfabe437954c2ced05a16fe023b724b5ca/packages/xo-web/src/xo-app/backup-ng/new/_schedules/new.js#L43-L45